persistence-jdbc/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/jdbc/JdbcPersistenceComponents.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.scaladsl.persistence.jdbc
+
+import scala.concurrent.ExecutionContext
+
+import akka.actor.ActorSystem
+import com.lightbend.lagom.internal.persistence.jdbc.SlickProvider
+import com.lightbend.lagom.internal.scaladsl.persistence.ReadSideConfig
+import com.lightbend.lagom.internal.scaladsl.persistence.jdbc.JdbcOffsetStore
+import com.lightbend.lagom.internal.scaladsl.persistence.jdbc.JdbcReadSideImpl
+import com.lightbend.lagom.internal.scaladsl.persistence.jdbc.OffsetTableConfiguration
+import play.api.Configuration
+import play.api.db.DBApi
+import com.lightbend.lagom.internal.scaladsl.persistence.jdbc.JdbcPersistentEntityRegistry
+import com.lightbend.lagom.scaladsl.persistence.PersistentEntityRegistry
+import com.lightbend.lagom.internal.scaladsl.persistence.jdbc.JdbcSessionImpl
+import com.lightbend.lagom.internal.scaladsl.persistence.OffsetStore
+
+/**
+ * Persistence JDBC components (for compile-time injection).
+ */
+trait JdbcPersistenceComponents {
+
+  def system: ActorSystem
```

Comment:
I guess the naming convention is to use exactly the same name as the class name, so this should be `actorSystem`